tests/conftest.py

```
@@ -21,13 +21,16 @@
 # PYTEST ASYNCIO CONFIGURATION
 # ============================================================================
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
 def event_loop():
-    """Create event loop for async tests"""
+    """Create event loop for async tests (function scope for isolation)"""
     import asyncio
```

Ruff (0.14.10)

Redefinition of unused `asyncio` from line 4

Remove definition: `asyncio`

(F811)